DepotDownloader/Ansi.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Spectre.Console;
+
+namespace DepotDownloader;
+
+static class Ansi
+{
+    // https://conemu.github.io/en/AnsiEscapeCodes.html#ConEmu_specific_OSC
+    // https://learn.microsoft.com/en-us/windows/terminal/tutorials/progress-bar-sequences
+    public enum ProgressState
+    {
+        Hidden = 0,
+        Default = 1,
+        Error = 2,
+        Indeterminate = 3,
+        Warning = 4,
+    }
+
+    const string ESC = "\u001b";
```

Comment:
I think this is `\e` in modern C#

Reply:
it complains about string escaping being a preview feature or something, i changed these consts to chars.